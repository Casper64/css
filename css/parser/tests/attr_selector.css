a[] {}

a[href^"test"] {}

a[href=test] {}

a[href="#test" {}