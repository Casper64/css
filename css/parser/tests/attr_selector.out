css/parser/tests/attr_selector.css:1:2: error: expecting an identifier
    1 | a[] {}
      |  ^
    2 | 
    3 | a[href^"test"] {}
css/parser/tests/attr_selector.css:3:8: error: expecting "=" after the attribute match type
    1 | a[] {}
    2 | 
    3 | a[href^"test"] {}
      |        ~~~~~~
    4 | 
    5 | a[href=test] {}
css/parser/tests/attr_selector.css:7:16: error: expecting a "]" to close the attribute selector
    5 | a[href=test] {}
    6 | 
    7 | a[href="#test" {}
      |                ^